### DISCUSSION TITLE
Can't use GoBig or GoLatent, Index error

### DISCUSSION THREAD

**Discussion by ARDEACT** [hlky/stable-diffusion-webui#684] [General]
![unknown](https://user-images.githubusercontent.com/46211295/188520822-44a4590b-bd91-4177-a052-99b5d7976d01.png)
Followed the instructions 3 times, since last week. Latest commit. Not working. The proper forks were used as well.

**Comment by DasWookie** [comment]
Same here. Been banging my head on this rock thinking it was just me... seems not.

**Reply by ARDEACT** [reply]
Nevermind. It seems fixed on latest commit.
